impeller/entity/entity_unittests.cc

```diff
@@ -1579,6 +1579,35 @@ TEST_P(EntityTest, ColorMatrixFilter) {
   ASSERT_TRUE(OpenPlaygroundHere(callback));
 }
 
+TEST_P(EntityTest, ColorMatrixFilterOnlyShowGreen) {
```

Comment:
It's probably worth just dropping this playground since there are already other manual verification playgrounds for this filter.

Reply:
(I think the `ColorMatrixFilter` playground above this one should also be deleted, since `ColorMatrixFilterEditable` does the same thing but with more interactivity)

Reply:
Done.